config/routes.rb

```diff
@@ -9,5 +9,9 @@
     end
   end
   root "home#index"
-  resources :companies, only: %i(new create show)
+  resources :companies, only: %i( new create show edit update index) do 
```

Comment:
```suggestion
  resources :companies, only: %i(new create show edit update index) do 
```

Eu acho que alguns PRs vieram com `%i[bla ble bli]`, colchetes no lugar de parenteses. Não é um problema estar diferente, mas é bom ter um padrão.

Reply:
Estabeleceremos um padrão.